charts/rancher-eks-operator/1.0.801/Chart.yaml

```diff
@@ -4,5 +4,5 @@ description: A Helm chart for provisioning EKS clusters
 home: https://github.com/rancher/eks-operator
 sources:
   - "https://github.com/rancher/eks-operator"
-version: 1.0.800
+version: 1.0.801
```

Comment:
Are we going to tag a 1.0.9 for 2.5.10 ?

Reply:
No, I don't believe so.